src/stories/iconButton.stories.js

```diff
@@ -0,0 +1,42 @@
+import React from "react"
+import { action } from "@storybook/addon-actions"
+
+import Button from "../components/button"
+import CtaCall from "../../.storybook/icons/ctaCall"
+
+export default {
+  title: "Icon Button",
+  component: Button,
+  args: {
+    storyName: "Default",
```

Comment:
There's no need to specify `label` here ? I did it for the button component but it might not be the best way actually.